modules/@apostrophecms/doc-type/index.js

```diff
@@ -2688,6 +2669,7 @@ module.exports = {
             if (field.fieldsStorage) {
               add.push(field.fieldsStorage);
             }
+            add.push(field.relatedReverseIds);
```

Comment:
Not sure if this one is needed or not

Reply:
Not needed, and could be large.

Reply:
We'll probably use this in frontend eventually to show related documents but it would be in a context where we loaded the whole document without projections.